ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -47,7 +42,8 @@ export class LoggingService {
     }
 
     private stringify(obj: object): string {
-        return Object.keys(obj).map(key => `${key}: ${obj[key]}`).join(', ');
+        return Object.keys(obj).map(key => (typeof obj[key] === 'object') ? `${key}: ${JSON.stringify(obj[key])}` : `${key}: ${obj[key]}`)
```

Comment:
use `JSON.stringfy()` to decode `[object, object]`.

<img width="1564" alt="Screenshot 2023-08-25 at 5 07 29 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/9a38a3f6-4be8-4cc2-971f-a0bdceb15ce9">

Reply:
This ternary operation is pretty dense--maybe pull out `$obj[key]` into a separate var so it's a little easier on the eyes?

Reply:
Good idea. Thank you.